### PR TITLE
Feat/44 responsive design listitem remove middletext

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
Fixes stacking of middletext on mobilescreen along with updating DS and adding break-word to ActionBar

## Related Issue(s)
- #44 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export enum ListTextColor {
 export interface CompactDeletableListItemProps {
   removeCallback?: (() => void) | null;
   leftText: string;
-  middleText: string;
+  middleText?: string | boolean;
```
Comment (allinox) [inline src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx:19]: Why a boolean?
Does that mean I can feed the value `true` as a prop and expect a default middle text?

And if not, and this is only meant for the user of the component to be able to indicate they don't want any middle text, can't that just be achived by not passing a string at all? An undefined string is after all evaluated to false anyway.

File: src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx [hunk 2]
```diff
@@ -34,44 +34,88 @@ export const CompactDeletableListItem = ({
...
+          <div className={classes.deleteSection}>
+            {removeCallback && (
+              <Button
+                variant={ButtonVariant.Quiet}
+                color={ButtonColor.Danger}
+                icon={<MinusCircle />}
+                onClick={removeCallback}
+                aria-label={String(t('api_delegation.delete'))}
+              >
+                {!isSm && t('api_delegation.delete')}
```
Comment (allinox) [inline src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx:72]: This check of !Sm happens inside another check for Sm so won't it always be false?

File: src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx [hunk 3]
```diff
@@ -34,44 +34,88 @@ export const CompactDeletableListItem = ({
...
+              </Button>
+            )}
+          </div>
+        </div>
+      ) : (
```
Comment (allinox) [inline src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx:77]: These two conditional sections have a lot of duplicate code. If checking for the presence of middle text is the only difference between the two, then can't the check for a small screen happen only there?

File: src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.module.css [hunk 4]
```diff
@@ -1,45 +1,100 @@
...
-  margin-right: 10px;
+  .listItemTexts{
+    flex: 1;
+    display: flex;
+    font-size: 16px;
+    --list-text-color-error: #E02E49;
+    align-items: center;
+  }
+
+  .baseListItemContent{
```
Comment (allinox) [inline src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.module.css:59]: Looks like there's a lot of duplicate code between the two screen sizes, like this baseListItemContent for example, which seems to be the exact same in both cases.
Maybe only the code affected by the change in screen size should be defined separately for the two cases?

File: src/components/reusables/SummaryPage/SummaryPage.tsx [hunk 5]
```diff
@@ -1,22 +1,22 @@
-import { Panel, PanelVariant } from '@altinn/altinn-design-system';
+import {
+  Page,
+  PageContent,
+  PageHeader,
```
Comment (allinox) [inline src/components/reusables/SummaryPage/SummaryPage.tsx:4]: Are we going back to the page from the design system or is this a case of git acting up? 😅

**Review by allinox — CHANGES_REQUESTED** [review]
I haven't actually tested this, so I don't know how it looks visually speaking. 😅 I've just taken a quick look at the code.

File: src/components/reusables/SummaryPage/SummaryPage.tsx (hunk 5 above)
Comment (Albertlarsen) [reply src/components/reusables/SummaryPage/SummaryPage.tsx]: git acting up

**Review by allinox — APPROVED** [review]
Nice 😊

**Pull request merged**